web/gui/dashboard_info.js

```diff
@@ -339,6 +339,12 @@ netdataDashboard.menu = {
         info: 'Performance metrics for <b>PostgreSQL</b>, the open source object-relational database management system (ORDBMS).'
     },
 
+    'pgbouncer': {
+        title: 'PgBouncer',
+        icon: '<i class="fas fa-exchange-alt"></i>',
+        info: 'Performance metrics for PgBouncer, the open source connection pooler for PostgreSQL.'
```

Comment:
Personally I would use `the open source connection pooler` for PgBouncer without any problems, because I worked with it during years, but it is not the unique, I suggest to change for `an open source connection pooler`.

Reply:
After our meeting I am resolving this and I will push a commit.